proto/event.proto

```
@@ -0,0 +1,28 @@
+
```

Buf (1.47.2)

Files must have a package defined.

(PACKAGE_DEFINED)